### PR TITLE
Dashboard/login set up

### PR DESCRIPTION
**Pull request by denced**
The main page is mostly functional and configured UI @bronsonmullens @alext8900 @johnnydhicks. We need to create a favorites view and redirect after okta login

### CODE REVIEW

**Review by bronsonmullens — APPROVED** [review]
I'm really liking the buttons on the dashboard. Good work with the shadows! We'll just need to remove those print debugs once we're ready to deploy it. Confirmed that signing in still works.

File: labs-ios-starter/Map/MapViewController.swift [hunk 1]
```diff
@@ -36,10 +36,10 @@ class MapViewController: UIViewController, MKMapViewDelegate {
         
         mapView.delegate = self
         addUserTrackingButton()
-        addSearbarTable()
+        addSearchbarTable()
     }
     
-    func addSearbarTable() {
+    func addSearchbarTable() {
```
Comment (alext8900) [inline labs-ios-starter/Map/MapViewController.swift:42]: finally fixed the spelling :) Good job

File: labs-ios-starter/LoginNeumorphicView.swift [hunk 2]
```diff
@@ -35,54 +37,49 @@ class LoginNeumorphicView: UIView {
...
-        lightShadow.shadowOffset = CGSize(width: -5, height: -5)
+        lightShadow.cornerRadius = ButtonRadius.cornerRadius
+        lightShadow.backgroundColor = ButtonRadius.lightBackgroundColor
+        lightShadow.shadowColor = ButtonRadius.lightShadowColor
+        lightShadow.shadowOffset = ButtonRadius.shadowOffsetTapped
         lightShadow.shadowOpacity = 1
-        lightShadow.shadowRadius = 15
+        lightShadow.shadowRadius = ButtonRadius.cornerRadius
         self.layer.insertSublayer(lightShadow, at: 0)
     }
```
Comment (alext8900) [inline labs-ios-starter/LoginNeumorphicView.swift:84]: maybe we should try and find a way to make this easier with less code hmmmm

**Review by alext8900 — APPROVED** [review]
Great job man.

**Pull request merged**